tests/e2e/linalg_ext_ops/softmax.mlir

```diff
@@ -0,0 +1,14 @@
+func.func @softmax() {
+  %input = util.unfoldable_constant dense<1.0> : tensor<2x8x4xf32>
+
+  %init = tensor.empty() : tensor<2x8x4xf32>
+  %1 = iree_linalg_ext.softmax dimension(2)
+       ins(%input : tensor<2x8x4xf32>)
+       outs(%init : tensor<2x8x4xf32>) -> tensor<2x8x4xf32>
+  %2 = flow.tensor.reshape %1 : tensor<2x8x4xf32> -> tensor<2x8x4xf32>
```

Comment:
why do you need this line?

Reply:
Good catch! That was an artifact from another test and is definitely not required. Will remove.